### PR TITLE
[DRAFT] Implement one-init

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This is yet another one-init, which works like the following step:

- generate cfg file for import, optimize, quantization
- ask backend-init to update and save cfg file
- add some more comment

ONE-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
This is a modified version of #9421.

File: compiler/one-cmds/dummy-driver/src/dummy-init.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * ... json output
+ *
+ */
+
+#include <iostream>
+
+const char *output =
+  R"(
+{
+  \"workflow\": [\"one-import\", \"one-optimize\", \"one-quantize\", \"one-codegen\"],
```
Comment (seanshpark) [inline compiler/one-cmds/dummy-driver/src/dummy-init.cpp:31]: I've searched for this `R("...")`, raw string and looks good feature.
And I think we don't need to use `\"` but just `"` seems OK.

File: compiler/one-cmds/dummy-init [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#/bin/bash
+
+cat << EOF
+{
+    "workflow": ["one-import", "one-optimize", "one-quantize", "one-codegen"],
+    "one-optmize": {
```
Comment (seanshpark) [inline compiler/one-cmds/dummy-init:6]: typo `optmize`

File: compiler/one-cmds/one-init [hunk 3]
```diff
@@ -26,12 +26,77 @@ import itertools
...
 import sys
+import configparser
 
 import configparser
 import utils as _utils
 
 # TODO Find better way to suppress trackback on error
 sys.tracebacklimit = 0
+'''
+NOTE
```
Comment (seanshpark) [inline compiler/one-cmds/one-init:37]: how about `NOTE about backend specific init`?

**Comment by yunjayh** [comment]
I'm going to inherit his work about `one-init`. Let me close and open a new draft and prs. (I don't have write permission to his repositories.) 

Thank you for your effort. :smile: :+1:

**Pull request closed without merging**